app/views/annotator/index.html.haml

```diff
@@ -20,7 +20,7 @@
               = t('annotator.options')
             .prefrences
               .chips
-                = render(ChipsComponent.new(name: 'whole_word_only', label: t('annotator.whole_word_only'), checked: params[:whole_word_only] || true))
+                = render(ChipsComponent.new(name: 'whole_word_only', label: t('annotator.whole_word_only'), checked: @init_whole_word_only.nil? || params[:whole_word_only]))
```

Comment:
why this solution ? and  no just have  `checked: params[:whole_word_only].to_s.eql('true')`

Reply:
We want it to be selected by default

Reply:
yes the question, was the need to have a new variable for that?